pkgs/development/compilers/elm/packages/elm-format.nix

```
@@ -1,40 +1,40 @@
-{ mkDerivation, fetchgit, ansi-terminal, ansi-wl-pprint, array, base, binary
-, bytestring, cmark, containers, directory, filepath, free, HUnit
-, indents, json, mtl, optparse-applicative, parsec, process
-, QuickCheck, quickcheck-io, split, lib, tasty, tasty-golden
-, tasty-hunit, tasty-quickcheck, text
+{ mkDerivation, ansi-wl-pprint, avh4-lib, base, bimap, cmark
+, containers, elm-format-lib, elm-format-test-lib, fetchgit, json
+, lib, mtl, optparse-applicative, parsec, QuickCheck, quickcheck-io
+, relude, tasty, tasty-hspec, tasty-hunit, tasty-quickcheck, text
 }:
 mkDerivation {
```

we can use `rec` here for recursive...

```suggestion
mkDerivation rec {
```